source/common/upstream/cluster_manager_impl.cc

```diff
@@ -73,8 +73,8 @@ void ClusterManagerInitHelper::removeCluster(Cluster& cluster) {
   // It is possible that the cluster we are removing has already been initialized, and is not
   // present in the initializer list. If so, this is fine.
   cluster_list->remove(&cluster);
-  ENVOY_LOG(info, "cm init: removing: cluster={} primary={} secondary={}", cluster.info()->name(),
-            primary_init_clusters_.size(), secondary_init_clusters_.size());
+  ENVOY_LOG(info, "cm init: init complete: cluster={} primary={} secondary={}",
```

Comment:
Personally, I found "removing" to be confusing when I was debugging this. It seemed like the cluster was being removed. Open to other wordings?